model-server-openapi/specifications/model-server.yaml

```diff
@@ -291,6 +291,25 @@ paths:
           $ref: '#/components/responses/200json'
         "500":
           $ref: '#/components/responses/500'
+  /v2/repositories/{repository}/branches/{branch}/query/warmup:
+    get:
+      operationId: postRepositoryBranchQueryWarmup
```

Comment:
post in the operation ID for a get route does not make much sense

Reply:
Changed to `getRepositoryBranchQueryWarmup`.